test/distributed/checkpoint/fsdp/test_fsdp_dsd.py

```diff
@@ -458,39 +458,37 @@ def _test_save_with_fsdp2_tp_and_load_with_tp(
         """
         Test that we can save a model with FSDP2 + TP on 2d mesh and load it with TP.
         """
+        mlp_dim = 5
 
-        def _get_base_model(mlp_dim: int = 2):
-            base_model = nn.Sequential(MLP(mlp_dim), MLP(mlp_dim), MLP(mlp_dim))
+        def _get_base_model(mlp_dim: int = mlp_dim):
+            base_model = nn.Sequential(
+                MLP(mlp_dim, dim_multiplier=1),
+                MLP(mlp_dim, dim_multiplier=1),
+                MLP(mlp_dim, dim_multiplier=1),
+            )
```

Comment:
any reason we change the model hyper-parameter instead of passing different `mlp_dim` value to test uneven sharding?

Reply:
do you mean why do i change the 'dim_multiplier' instead of just changing the mlp_dim? or ?

Reply:
right. the default value of `dim_multiplier` is 4. Why are we changing the value here? Is it to create an uneven sharding case? Then we better keep the `mlp_dim=2, dim_multiplier=4` case by parameterizing `dim_multiplier` in test in `[1, 4]`. WDYT? We can add it in another PR to avoid wasting CI.